examples/tree_1d_dgsem/elixir_shallowwater_well_balanced_wet_dry.jl

```diff
@@ -0,0 +1,100 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# Semidiscretization of the shallow water equations
+
+equations = ShallowWaterEquations1D(gravity_constant=9.812)
+cfl = .3
+
+function initial_condition_well_balanced_CN(x, t, equations:: ShallowWaterEquations1D)
```

Comment:
```suggestion
function initial_condition_well_balanced_chen_noelle(x, t, equations:: ShallowWaterEquations1D)
```
Add a docstring with a reference and DOI to the Chen and Noelle paper where this initial condition comes from.